updating/index.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, check the following information:
+
+* xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[About the OpenShift Update Service]
+* xref:../updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
```

Comment:
And also this link is not available.
https://github.com/openshift/openshift-docs/blob/aa56c230747b39f3b52ae044e3e876e399e96d4c/updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service

Reply:
The link in the preview is working fine now

Reply:
@xenolinux When I click [Understanding support policy for unmanaged Operators] in https://github.com/openshift/openshift-docs/blob/aa56c230747b39f3b52ae044e3e876e399e96d4c/updating/index.adoc, it still return me with 404.

Reply:
I fixed the issue. Can you please check now @jiajliu